dallinger/recruiters.py

```diff
@@ -205,23 +279,57 @@ def prolific_submission_listener():
 PROLIFIC_AD_QUERYSTRING = "&PROLIFIC_PID={{%PROLIFIC_PID%}}&STUDY_ID={{%STUDY_ID%}}&SESSION_ID={{%SESSION_ID%}}"
 
 
+def _current_hits(service, app):
+    if app is not None:
+        if type(service) is MTurkService:
```

Comment:
This looks like code that should be refactored into the `Service` classes?

Reply:
Done